NEWS.md

```diff
@@ -1,6 +1,19 @@
 # NEWS for Lrama
 
-## Lrama 0.7.0 (2025-XX-XX)
+## Lrama 0.7.0 (2025-01-18)
+
+## Support the generation of the IELR(1) parser described in this paper
```

Comment:
Could you add `[EXPERIMENTAL]` to the IELR introduction?

Reply:
Add [EXPERIMENTAL]